src/subscript/sector2fluxnum/completions.py

```diff
@@ -0,0 +1,44 @@
+from ecl2df import compdat, EclFiles
+
+
+def generate_compdat_dataframe(ECL_DATA_file_name):
+    """
+    Create a datafram of unrolled well completions
+
+    Args:
+    Input DATA file name
+
+    Returns:
+    dataFrame with the following header:
+
+    """
+    ECL_file = EclFiles(ECL_DATA_file_name)
+    compdat_df = compdat.df(ECL_file)
+
+    return compdat_df
+
+
+def get_completion_list(dframe):
+    """
+    Create a datafram of unrolled well completions
+
+    Args:
+    Pandas data frame with completions
```

Comment:
PS: This probably does not format well in the html-generated API docs, but that itself is not a requirement. Check the docs by running `python setup.py build_sphinx` and then open the generated html.

Reply:
I have checked. It formats ok.